ironfish-cli/src/commands/wallet/transactions/index.ts

```diff
@@ -62,6 +62,14 @@ export class TransactionsCommand extends IronfishCommand {
       options: ['notes', 'transactions', 'transfers'],
       helpGroup: 'OUTPUT',
     }),
+    'filter.start': Flags.string({
+      description: 'include transactions after this date (inclusive). Example: 04/20/2023',
+      parse: (input) => Promise.resolve(new Date(input).toISOString()),
```

Comment:
`input` gets converted to a Date, then to an ISO string, then back to a Date, then to an integer. I wonder if we could just remove the `.toISOString()` here and have this `parse` function return a Date?

Reply:
You need to create a new custom flag for this that supports dates. I can do this but I'll need to open a separate Pr that will then block this. I can start working on that now but it's outside the goal of this individual PR.

Reply:
I'll work on adding this

Reply:
No need to add this functionality as far as I'm concerned--I was just wondering if it was possible. I was assuming that `parse` could return any type